content/concepts/consensus-network/invariant-checking.md

```diff
@@ -0,0 +1,152 @@
+# Invariant Checking
+
+This article provides a high-level overview of invariant checking, why it exist, how it works, and lists active invariants.
```

Comment:
I feel this sentence is unnecessary. The table of contents tells the user the same information. We can cut to the chase and start describing invariant checking right away.